README.md

```diff
@@ -132,6 +132,130 @@ PrestoGateway records history of recent queries and displays links to check quer
 The Gateway admin page is used to configure the gateway to multiple backends. Existing backend information can also be modified using the same.
 ![prestogateway.lyft.com/entity](/docs/assets/prestogateway_ha_admin.png)
 
+## Resource Groups API
+
+### Add a resource group
+To add a single resource group, specify all relevant fields in the body.
+```$xslt
+curl -X POST http://localhost:8080/presto/resourcegroup/create \
+ -d '{  "resourceGroupId": 1, \
+        "name": "resourcegroup1", \
+        "softMemoryLimit": "100%", \
+        "maxQueued": 100, \
+        "softConcurrencyLimit": 100, \
+        "hardConcurrencyLimit": 100, \
+        "environment": "test", \
+        "schedulingPolicy": null, \
+        "schedulingWeight": null, \
+        "jmxExport": null, \
+        "softCpuLimit": null, \
+        "hardCpuLimit": null, \
+        "parent": null, \
+        "environment": "test" \
+    }'
+```
+
+### Get existing resource group(s)
+If no resourceGroupId (type long) is specified, then all existing resource groups are fetched. 
+```$xslt
+curl -X GET http://localhost:8080/presto/resourcegroup/read/{INSERT_ID_HERE}
+```
+
+### Update a resource group
+Specify all fields in the body, which will overwrite the existing resource group's properties.
```

Comment:
it will overwrite the properties for this specific `resourceGroupId` only, correct?

Reply:
Oh yes, let me make that more clear.